test/test_data.rb

```diff
@@ -287,7 +287,7 @@ def make_subscription_array(customer_id)
       {
         :data => [make_subscription, make_subscription, make_subscription],
         :object => 'list',
-        :resource_url => '/v1/customers/' + customer_id + '/subscriptions'
+        :resource_url => '/v1/subscriptions'
```

Comment:
Similar to the stripe-python bindings, we'll need to functions here: `make_customer_subscription_array` and `make_subscription_array`. The only difference is the URL.